samples/dotnet/kernel-syntax-examples/Example21_ChatGPTPlugins.cs

```diff
@@ -19,7 +19,7 @@ private static async Task RunChatGptPluginAsync()
     {
         var kernel = new KernelBuilder().WithLogger(ConsoleLogger.Log).Build();
         using HttpClient importHttpClient = new HttpClient();
-        importHttpClient.DefaultRequestHeaders.Add("User-Agent", "Microsoft.SemanticKernel");
+        importHttpClient.DefaultRequestHeaders.Add("User-Agent", "Microsoft-Semantic-Kernel");
```

Comment:
Reviewers note: spoke with @adrianwyatt about this. Technically not related to the lang version change, but confirmed the reason. I checked other uses of the User-Agent throughout the repo and confirmed this is just making it match what we have elsewhere (see https://github.com/microsoft/semantic-kernel/blob/05c586b9d8c040e33b47630c969495b2d7facd8b/dotnet/src/Connectors/Connectors.AI.OpenAI/CustomClient/OpenAIClientBase.cs#L189). Approved.